docker-entrypoint.sh

```diff
@@ -1,8 +1,13 @@
 #!/bin/bash
 
-# Add crontab jobs
-echo "Calling script to set up server cronjobs"
-/bin/bash scripts/cron_setup.sh
+if [[ "$USE_CRON" == "true" ]]; then
+    # Add crontab jobs
+    echo "Calling script to set up server cronjobs"
+    /bin/bash scripts/cron_setup.sh
+else
+    echo "Skipping cron setup"
+fi
+
```

Comment:
Mechanism to allow local development to use the cron to execute `rake check:all`